CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 1.0.3
```

Comment:
Can you change this to `# Unreleased`? I will update it when I bump the version in the cabal file for release to Hackage; I want to get a few other small tweaks in for a patch release.

Reply:
Done!